openslides/agenda/static/templates/agenda/item-list.html

```diff
@@ -8,11 +8,14 @@
         <translate>New</translate>
       </a>
       <!-- import -->
-      <a ui-sref="topics.topic.import" os-perms="agenda.can_manage"
+      <span os-perms="agenda.can_manage">
```

Comment:
> Is it useful that a person with 'can_manage' does not have 'can_see_hidden_items'?

No, it's not useful. But we need the option to set 'can_see_hidden_items' permission for normal users who should see more than official agenda items. So I can accept it that staff users have to consider that they have also this extra permission for internal items. No changes required for me.